benches/src/util.js

```diff
@@ -17,6 +17,27 @@ export function afterFrameAsync() {
 	return promise;
 }
 
+export function getRowIdSel(index) {
```

Comment:
I put these `getRowIdSel` and `getRowLinkSel`  functions in the util file cuz if we add more benches from js-framework-benchmark, they'll be used by multiple benches